streampark-console/streampark-console-service/src/main/java/org/apache/streampark/console/system/service/impl/UserServiceImpl.java

```diff
@@ -159,9 +160,14 @@ public void resetPassword(String[] usernames) {
      * @return permissions
      */
     @Override
-    public Set<String> getPermissions(String username) {
-        List<Menu> permissionList = this.menuService.findUserPermissions(username);
-        return permissionList.stream().map(Menu::getPerms).collect(Collectors.toSet());
+    public Set<String> getAllTeamPermissions(String username) {
```

Comment:
`getPermissions` method can replaced `getAllTeamPermissions` method, we don't need this method, What do you think?